src/map/skill.cpp

```diff
@@ -10653,8 +10714,12 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 	case MH_GOLDENE_FERSE:
 		clif_skill_nodamage(src,bl,skill_id,skill_lv,
 			sc_start(src,bl,type,100,skill_lv,skill_get_time(skill_id,skill_lv)));
-		if (hd)
-			skill_blockhomun_start(hd, skill_id, skill_get_time2(skill_id,skill_lv));
+		if (hd != nullptr)
+		{
+			skill_blockhomun_start(hd, skill_id, skill_get_time2(skill_id, skill_lv));
+			if (hd->master != nullptr)
+				skill_blockpc_start(hd->master, skill_id, skill_get_time2(skill_id, skill_lv));
```

Comment:
```suggestion
			skill_blockhomun_start(hd, skill_id, skill_lv);
```
Another `skill_get_time2()` reference.